tests/unit/session/test_parallel.py

```diff
@@ -51,6 +51,28 @@ def test_parallel_live(cmd, initproj):
     result.assert_success()
 
 
+def test_parallel_nospinner(cmd, initproj):
+    initproj(
```

Comment:
do add that env var you've seen over at the test I've pointed to, no need to actually create the venv vars here; also I would suggest add some sleep (0.2s enough) as part of the command to allow the spinner to trigger

Reply:
If you look at that output at the bottom of the help you have all env-vars clearly documented...

Reply:
Sorry here we clearly switched from "out of my comfort zone but I will manage" to "I don't understand anymore and I can't provide the requested work".

Please feel free to decline this PR and rewrite it a better way, I will read it and learn a lot for the next time. No offense, really, I already leaned a lot today and abused to much of your time.